io_export_objex2/properties.py

```diff
@@ -374,6 +416,19 @@ class ObjexMaterialCollisionProperties(bpy.types.PropertyGroup):
             default=False
         )
 
+def omp_change_alpha(self, context):
```

Comment:
I'll probably want the name to be changed eventually but in the mean time can you add a comment for what does "omp" stand for?